LICENSE

```diff
@@ -0,0 +1,11 @@
+Copyright (C) <2022> Diego Alejandro Cortés Roa
```

Comment:
As Manuel Sugested, this seems to be the wrong template. Check the one he left on the comments